ledger/acctonline.go

```diff
@@ -504,11 +504,22 @@ func (ao *onlineAccounts) postCommit(ctx context.Context, dcc *deferredCommitCon
 func (ao *onlineAccounts) postCommitUnlocked(ctx context.Context, dcc *deferredCommitContext) {
 }
 
-// OnlineTotals return the total online balance for the given round.
-func (ao *onlineAccounts) OnlineTotals(rnd basics.Round) (basics.MicroAlgos, error) {
+// onlineTotals return the total online balance for the given round.
+func (ao *onlineAccounts) onlineTotals(rnd basics.Round) (basics.MicroAlgos, error) {
 	ao.accountsMu.RLock()
-	defer ao.accountsMu.RUnlock()
-	return ao.onlineTotalsImpl(rnd)
+	totalsOnline, err := ao.onlineTotalsImpl(rnd)
+	ao.accountsMu.RUnlock()
+	if err == nil {
+		return totalsOnline, err
+	}
+
+	var roundOffsetError *RoundOffsetError
+	if !errors.As(err, &roundOffsetError) {
+		return basics.MicroAlgos{}, err
+	}
```

Comment:
this should not be checked. `roundParamsOffset` only returns errors related to the fact the round requested out of the in-memory range: both RoundOffsetError and another error are about the same fact.